README.md

```diff
@@ -1,2 +1,307 @@
+
+### gOTU table generation
+
+To generate a gOTU table, one needs a multiplexed alignment file, or a directory of per-sample alignment files. These files can be generated by aligning sequencing data against a reference genome database. We recommend using [**SHOGUN**](https://github.com/knights-lab/SHOGUN) with the "Web of Life" database (**WoL**, available for download at: https://biocore.github.io/wol/). For example:
+
+```bash
+shogun align -a bowtie2 -d WoLr1 -i input.fasta -o .
+```
+
+Then one can run Woltka to convert the alignment file(s) into a gOTU table:
+
+```bash
+woltka gotu -i alignment.bowtie2.sam -o table.tsv
+```
+
+The output file `table.tsv` is a table with rows as genome IDs (gOTUs), columns as sample IDs, and cell values as counts of gOTUs in samples.
+
+Note: Both SHOGUN and WoL are available at the [**Qiita**](https://qiita.ucsd.edu/) server. If you are a Qiita user, the alignment file can be automatically generated and downloaded from the Qiita interface.
+
+### gOTU analysis using QIIME 2
+
+With QIIME, the gOTU table can be converted into BIOM format:
+
+```bash
+biom convert --to-hdf5 -i table.tsv -o table.biom
+```
+
+The BIOM table can be imported into a QIIME artifact:
+
+```bash
+qiime tools import --type FeatureTable[Frequency] --input-path table.biom --output-path table.qza
+```
+
+These intermediate steps are automated if you use the [QIIME 2 plugin of Woltka](woltka/q2).
+
+One can then investigate the microbiome by applying classical QIIME analyses on the gOTU table. For example, with the [WoL reference phylogeny](https://biocore.github.io/wol/), one can do:
+
+```bash
+qiime diversity core-metrics-phylogenetic \
+  --i-phylogeny tree.qza \
+  --i-table table.qza \
+  --p-sampling-depth 1000 \
+  --m-metadata-file metadata.tsv \
+  --output-dir .
+```
+
+### Alignment ambiguity
+
+It is quite common that one query sequence can be aligned to multiple reference genomes. In such cases, Woltka by default counts each gOTU as 1 / _k_, where _k_ is the total number of matching genomes.
+
+Alternatively, one may choose to discard all non-unique matches, by adding a flag:
+
+```bash
+woltka gotu --no-ambig ...
+```
+
+### Custom alignment
+
+Technically, one can use any sequence aligners and reference genome databases to generate alignment files which can then be converted into a gOTU table. We cannot validate the goodness of outcome, but understand that you may have this intention considering the consistency with existing parts of your analytical pipeline. For examples:
+
+```bash
+bwa mem refseq.fna input.R1.fq input.R2.fq > output.sam
+```
+
+```bash
+blastn -db refseq_genomes -query input.fa -max_target_seqs 1 -outfmt 6 -out output.txt
+```
+
+However, most of these protocols generate mappings of reads to nucleotides (e.g., chromosomes or scaffolds), rather than to genomes. In order to produce gOTUs, one needs to supply Woltka with a nucleotide-to-genome mapping file:
+
+```bash
+woltka gotu --map nucl2g.txt ...
```

Comment:
this nucl2g.txt may deserve more background or a reference.

Reply:
Added example file.